### PR TITLE
Revert "Revert "Combine strings and variables: draw_turn block""

### PR DESCRIPTION
**Pull request by nkiruka**
Reverts code-dot-org/code-dot-org#30668

New constants (TURN_BY_DIRECTION_VALUES and TURN_DIRECTION_VALUES) are added so variants of the draw_turn block reference the correct strings to ensure that the correct text is shown in the field dropdown. Follow-up PR to use the interpolateMsg function in "draw_turn"  blocks.

### CODE REVIEW

File: apps/i18n/turtle/en_us.json [hunk 1]
```diff
@@ -90,6 +90,8 @@
   "sizeParameter": "size",
   "step": "step",
   "tooFewColours": "You need to use at least %1 different colors for this puzzle.  You used only %2.",
+  "turnLeft": "turn left by",
```
Comment (bethanyaconnor) [inline apps/i18n/turtle/en_us.json:93]: nit: can these keys be `turnLeftBy` and `turnRightBy`? It would be consistent with the variables in `blocks.js`.

Comment (nkiruka) [reply apps/i18n/turtle/en_us.json:93]: Good observation.  For the interim, it is "turnLeft" & "turnRight" as these are the strings that have translations in Crowdin, prior to the changes I made.  The follow-up PR (i.e to update all the draw_turn blocks with interpolateMsg) will only reference the "left" and "right" strings.

Comment (bethanyaconnor) [reply apps/i18n/turtle/en_us.json:93]: Ah gotcha. Thanks for the explanation!

**Review by Hamms — APPROVED** [review]
LGTM! Is the work to update the other blocks to use the new values tracked somewhere?

**Comment by nkiruka** [comment]
@Hamms Good point!  I will add a jira item

**Pull request merged**